### ISSUE TITLE
ops(release): launch PPO baseline training on alcubierre cluster for #371 frozen baselines

### ISSUE THREAD

**Issue by rjwalters** [rjwalters/bucket-brigade#384]
**Parent**: #371 (Frozen baseline release, slice 3/5 of #365)

## Why filed

#371's acceptance includes "Trained PPO baseline checkpoints per scenario (from M2.1 / #360) with reported scores in a \`scores.json\`". #360 is currently \`loom:blocked\` because the upstream Nash phase diagram (#358 → now Run 2a in #381) hasn't produced its results. But the *operator-driven* part — actually running PPO on the alcubierre cluster — is a concrete launchable task that doesn't require waiting on the full #360 ramp. This issue captures that subset for execution as soon as a stable training config is available.

## Goal

Run PPO baseline training across the canonical scenario set on the alcubierre cluster (\`\$COMPUTE_HOST_CLUSTER\`), producing per-scenario checkpoints and a \`scores.json\` ready to be wrapped by #371's loader API.

## Scenarios to train (from the canonical scenario registry shipped in slice 1 / #369)

Reference: \`bucket_brigade/envs/scenarios_generated.py\` and its \`_SCENARIO_REGISTRY\` dict. Suggested initial set:

- \`minimal_specialization-v1\` (canonical P3 substrate)
- \`default-v1\`
- \`positional_default-v1\`
- \`chain_reaction-v1\`
- \`trivial_cooperation-v1\`
- \`v2_minimal-v1\` (2 houses × 4 agents)

Skip scenarios the paper doesn't reference in headline numbers — verify the final list against the paper outline before launching.

## Operator workflow (manual — \`loom:curated\`, not \`loom:issue\`)

1. \`source .env && ssh \"\$COMPUTE_HOST_CLUSTER\"\` (alcubierre)
2. \`cd ~/GitHub/bucket-brigade && git pull && bash bucket-brigade-core/build.sh\`
3. \`tmux new -s ppo-baselines\`
4. Per scenario, launch a training run with the project's standard PPO config. Reference command (verify exact flags against \`experiments/p3_specialization/train.py\` argparse before running):
   \`\`\`bash
   uv run python -m experiments.p3_specialization.train \\
     --scenario \$SCENARIO --seeds 42 43 44 \\
     --num-iterations 50 --output-root experiments/p3_specialization/baselines/\$SCENARIO/ \\
     2>&1 | tee logs/ppo_\${SCENARIO}_\$(date +%Y%m%d_%H%M%S).log
   \`\`\`
5. Detach + monitor periodically. Estimated wall clock per scenario: ~30 min – 2h GPU-time × 3 seeds (see #360 compute estimate).
6. On completion, package the best checkpoint per scenario into \`bucket_brigade/baselines/release/ppo/\$SCENARIO/checkpoint.pt\` with a sibling \`config.json\` capturing training hyperparameters and gap_closed score.
7. Author \`bucket_brigade/baselines/release/ppo/scores.json\` summarizing per-scenario {seed, gap_closed, config_hash}.
8. Open a results-only PR with \`loom:review-requested\` and \`Refs #371\` (don't close #371 — that's bigger than this run).

## Compute estimate

- 6 scenarios × 3 seeds × ~1h average = ~18 GPU-hours on alcubierre

## Acceptance criteria

- [ ] Per-scenario checkpoint in \`bucket_brigade/baselines/release/ppo/\$SCENARIO/checkpoint.pt\`
- [ ] \`scores.json\` lists every trained scenario with gap_closed value + training config hash
- [ ] Loader API (to be added by #371) can deserialize and run the checkpoints in a smoke test
- [ ] Results-only PR merged; #371 Acceptance "trained PPO baseline checkpoints" box ticked

## Out of scope

- Loader API itself (that lives in #371; this issue only produces the artifacts)
- HuggingFace upload (separate slice — #373)
- Full #360 trainability sweep across the NE phase diagram (this is the *baseline* checkpoints for the release, not the full sweep — those are separate runs once #381 Run 2 results land)

## References

- Parent: #371
- Sibling upstream (blocked): #360 (full PPO trainability sweep)
- Slice-1 dependency (closed, merged): #369 (env API + scenario registry)
- Scenario registry source: \`bucket_brigade/envs/scenarios_generated.py:_SCENARIO_REGISTRY\`
- Training script: \`experiments/p3_specialization/train.py\`

## Why filed now

Surfaced during the 2026-06-05 sweep. User has alcubierre cluster access; making the PPO baseline run a discrete scheduleable item turns #371 from "blocked indefinitely on #360" into "blocked on operator scheduling cluster time + Nash results landing".
